examples/3d-tiles/app.js

```diff
@@ -18,14 +18,29 @@ const INDEX_FILE = `${DATA_URI}/modules/3d-tiles/test/data/index.json`;
 const MAPBOX_TOKEN = process.env.MapboxAccessToken; // eslint-disable-line
 const MAPBOX_STYLE = 'mapbox://styles/mapbox/light-v9';
 
+const ADDITIONAL_EXAMPLES = {
+  name: 'additional',
+  examples: {
+    royalExhibitionBuilding: {
+      tilesetUrl:
+        'https://raw.githubusercontent.com/uber-common/deck.gl-data/master/3d-tiles/RoyalExhibitionBuilding/tileset.json',
+      coordinateSystem: COORDINATE_SYSTEM.METER_OFFSETS,
+      coordinateOrigin: [144.97212, -37.805177],
+      isWGS84: false,
+      depthLimit: 5,
+      color: [115, 101, 152, 200]
```

Comment:
Instead of overriding per example, we could define a DEFAULT_POINT_COLOR for all layers if they don't have color attributes.